### PR TITLE
[disc-cpu] Add a TensorFlow cpu inference demo on X86 platform

### PR DESCRIPTION
**Pull request by wyzero**

### CODE REVIEW

**Review by linearhit — CHANGES_REQUESTED** [review]

File: docs/tutorials/tensorflow_inference_and_training.md [hunk 1]
```diff
@@ -83,7 +87,78 @@ for batch in [2, 2, 4, 1, 1, 8, 8, 2, 16, 2]:
...
+batch-size.
+
+### Prologue: Download Frozen Model For X86 CPU
+
+```python
+!mkdir -p model
+!wget -P model http://pai-blade.oss-cn-zhangjiakou.aliyuncs.com/bladedisc_notebook_binaries/models/disc_bert_cpu_example/frozen.pb
+```
+
+### All You Need Are the Two Lines With CPU Specific Setting!
```
[the remark this answers is not in the document]
Comment (wyzero) [reply docs/tutorials/tensorflow_inference_and_training.md:107]: It's just comment！？

File: README.md [hunk 2]
```diff
@@ -60,22 +60,18 @@ training workloads.
 | Nvidia GPU |    Yes        |    Yes                 |    Yes               |
 | AMD GPU    |  Ongoing      |  Ongoing               |     No               |
 | Hygon DCU  |    Yes        |    Yes                 |    Yes               |
-|  X86       |    Yes        |  Not open-sourced yet [1]  |     No           |
-
-[1] The compute-intensive part of the X86 backend is already supported on the
-internal version. The code decoupling is ongoing and will be open-sourced soon,
-same for the end-to-end usability.
+|  X86       |    Yes        |    Yes                 |    Yes               |
```
[the remark this answers is not in the document]
Comment (wyzero) [reply README.md]: done, Thanks!

File: docs/tutorials/tensorflow_inference_and_training.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (wyzero) [reply docs/tutorials/tensorflow_inference_and_training.md]: done!

**Review by linearhit — APPROVED** [review]
LGTM

**Pull request merged**